multimodal/src/autogluon/multimodal/models/mmdet_image.py

```diff
@@ -41,7 +42,7 @@ def __init__(
         self,
         prefix: str,
         checkpoint_name: str,
-        num_classes: Optional[int] = None,
+        config_file: Optional[str] = None,
```

Comment:
For custom models, is it possible to not use `config_file` when loading, since it will be an extra hyperparameter that the user needs to add? For `timm_image` and `hf_text`, currently, we can reuse the `checkpoint_name` parameter and substitute name with checkpoint directory (which contains both weights and config).

Reply:
Sure, I changed the saved checkpoint name and config name to "model.pth" and "config.py". And if input `checkpoint_name` is a dir, it should include both checkpoint ("model.pth") and config ("config.py"). But I leave the `config_file` here in case we will use a checkpoint with different configs (e.g. yolox with mosaic v.s. yolox without mosaic).